osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -14,35 +14,39 @@ namespace osu.Game.Overlays.Settings.Sections.Input
 {
     public class MouseSettings : SettingsSubsection
     {
-        protected override string Header => "Mouse";
+        private readonly MouseHandler mouseHandler;
```

Comment:
This setting section uses `FrameworkSetting.MapAbsoluteInputToWindow`, which is obsoleted now - but I can't find a correspondent bindable on this mouse handler for some reason, despite what the obsoletion message says...?

Reply:
This needs to be re-implemented in the windows-specific raw input handler. Absolute input isn't actually supported by SDL2 itself. Not sure how this should be exposed to the user, if at all. We may just want to take a stance on it being always on or off, and see if anyone actually complains.

Keep in mind the primary purpose of this was for tablet users that wanted to choose whether their tablet's full area (assuming it outputs raw input HID) maps to the game window or to their standard desktop.